### PR TITLE
remove section saying static pods can't have health checks

### PR DESCRIPTION
**Pull request by relyt0925**
Static pods are able to have health checks in all the currently supported kubernetes versions

### CODE REVIEW

**Comment by sftim** [comment]
Hi @relyt0925
Would you like any help getting a contributor license agreement signed & recorded?

**Comment by relyt0925** [comment]
Hi @sftim I was trying to follow the docs... But if you have a super simple checklist to follow I would love it!

**Comment by relyt0925** [comment]
Working on that today/tonight

**Comment by relyt0925** [comment]
sent an email to `cla@cncf.io` with my signed agreement right now waiting to hear back on their response.

**Comment by RA489** [comment]
@relyt0925 You are still facing issue with cla things?

**Comment by relyt0925** [comment]
I haven't heard a response back from them yet. I sent them an email but the esign link they originally sent provides me nothing to actually esign to become a member. That's when I sent the scanned pdf to them.

**Comment by relyt0925** [comment]
@RA489

**Comment by relyt0925** [comment]
Ok got the esign link and signed it as of today. So it should progress soon.

**Comment by relyt0925** [comment]
@sftim @RA489 looks like it did register with the PR. Let me know if there are any other prereqs you need from me

**Comment by relyt0925** [comment]
@jsafrane could I get a review when you have a minute?

**Comment by relyt0925** [comment]
@stewart-yu could you give this a review when you get a minute

**Comment by relyt0925** [comment]
@stewart-yu just another bump

**Comment by relyt0925** [comment]
Hi @kbhawkey I saw you were active within the past couple of days. I think @stewart-yu might be on vacation

**Comment by relyt0925** [comment]
@sftim could you reach out to one of the members who has merge access when you get the chance? Otherwise I think this may just sit here. Thanks!

**Comment by sftim** [comment]
@relyt0925 if you wish, you can ask in [#sig-docs](https://app.slack.com/client/T09NY5SBT/C1J0BPD2M/) on Slack directly. If you're not yet part of that Slack workspace, you can [request](http://slack.kubernetes.io/) an invitation.

I'm minded that contributors and reviewers are mainly volunteers, so there's a limited bandwidth for people who are getting involved.

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-15956--kubernetes-io-master-staging.netlify.com/docs/tasks/configure-pod-container/static-pod/

**Comment by kbhawkey** [comment]
hello @relyt0925 . Just looking at the change and the context of the text.
Do you think it would be beneficial to readers to include a statement about _how_ or _what process_ (the kubelet?) checks the status of containers in a static Pod?

From v1.14 docs:
Static pods are managed directly by kubelet daemon on a specific node, without the API server observing it. It does not have an associated replication controller, and kubelet daemon itself watches it and restarts it when it crashes. There is no health check. Static pods are always bound to one kubelet daemon and always run on the same node with it.

The key phrase seems to be `without the API server observing it. `

**Comment by relyt0925** [comment]
@kbhawkey the health checking logic from what I experienced is no different than the health check logic of pods managed by the apiserver. Kubelet has the same health checking process for both. I think it could be confusing to try and draw any extra attention to that part. 

The restarts do register with the API server through the mirror pod if the kubelet is configured to talk to an API Server.

File: content/en/docs/tasks/configure-pod-container/static-pod.md [hunk 1]
```diff
@@ -15,7 +15,6 @@ observing them.
 Unlike Pods that are managed by the control plane (eg, a
 {{< glossary_tooltip text="Deployment" term_id="deployment" >}};
 instead, the kubelet watches each static Pod (and restarts it if it crashes).
-There are no health checks for the containers in a static Pod.
```
Comment (kbhawkey) [inline content/en/docs/tasks/configure-pod-container/static-pod.md]: @relyt0925 , Would you also fix a few nits:
- Line 16: There appears to be a missing closing parenthesis; also, could you substitute `for example` for `eg`.
Unlike Pods that are managed by the control plane (for example, a
{{< glossary_tooltip text="Deployment" term_id="deployment" >}}); instead, the kubelet watches each static Pod ...

The kubelet manages the lifecycle of static Pods.

- Line 21, The kubelet vs Kubelet

Comment (relyt0925) [reply content/en/docs/tasks/configure-pod-container/static-pod.md]: Yep I can!

**Comment by relyt0925** [comment]
@kbhawkey I made the changes you requested. Let me know if you need anything else

File: content/en/docs/tasks/configure-pod-container/static-pod.md (hunk 1 above)
Comment (kbhawkey) [reply content/en/docs/tasks/configure-pod-container/static-pod.md]: Thanks!
/lgtm

**Pull request merged**